pkg/policy/consumer.go

```diff
@@ -61,8 +61,8 @@ type Consumable struct {
 	Mutex sync.RWMutex
 	// Labels are the Identity of this consumable
 	Labels *Identity `json:"labels"`
-	// LabelList contains the same labels from identity in a form of a list, used for faster lookup
-	LabelList []*labels.Label `json:"-"`
+	// LabelArray contains the same labels from identity in a form of a list, used for faster lookup
+	LabelArray []*labels.Label `json:"-"`
```

Comment:
I think what @ianvernon meant in #838 was that `LabelList []*labels.Label` should be replaced with `LabelArray labels.LabelArray`